hail/python/hail/docs/genetics/index.rst

```diff
@@ -5,17 +5,15 @@ genetics
 .. toctree::
     :maxdepth: 2
 
-.. currentmodule:: hail.genetics
-
 .. rubric:: Classes
 
 .. autosummary::
     :nosignatures:
     :toctree: ./
     :template: class.rst
 
-    Call
-    Locus
-    Pedigree
-    ReferenceGenome
-    Trio
+    hail.Call
```

Comment:
This is the bit that fixes the `cannot import GroupedMatrixTable` issue. Obviously.

And it's not even our fault, just seem to be avoiding a sphinx bug

Reply:
I'm rather unhappy with removing current module in this one case. Does this mean everything else is silently waiting to break (e.g. linalg)?